### PR TITLE
brscan4: Add appstream metainfo

### PR DESCRIPTION
**Pull request by malfisya**
**Summary**

- Add appstream metainfo (Part of getsolus/packages#1389)
- Resolved getsolus/packages#3083

**Test Plan**

Verify metainfo with appstream-builder --packages-dir=. --include-failed -v

**Checklist**

- [x] Package was built and tested against unstable

### CODE REVIEW

File: packages/b/brscan4/files/brscan4.metainfo.xml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    </p>
+    <p>
+      Note that some portions are binary and provided for hardware enablement
+      so may not meet typical Solus packaging standards for paths.
+    </p>
+  </description>
+  <url type="homepage">https://global.brother</url>
+  <icon type="stock">brother</icon>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary:BROTHER</project_license>
```
Comment (malfisya) [inline packages/b/brscan4/files/brscan4.metainfo.xml:19]: Not sure about this

File: packages/b/brscan4/files/brscan4.metainfo.xml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  <developer_name>Brother Industries</developer_name>
+  <requires>
+    <kernel version="9.99" compare="eq">Linux</kernel>
+  </requires>
```
Comment (malfisya) [inline packages/b/brscan4/files/brscan4.metainfo.xml:23]: Is this required for all driver?

**Comment by malfisya** [comment]
I copied the metainfo from `nvidia-470-glx-driver`

File: packages/b/brscan4/files/brscan4.metainfo.xml (hunk 2 above)
Comment (joebonrichie) [reply packages/b/brscan4/files/brscan4.metainfo.xml:23]: Nah you can drop that. That only gets edited if the driver depends upon a specific kernel e.g. nvidia

**Review by joebonrichie — APPROVED** [review]

**Pull request merged**